template-tests.ps1

```diff
@@ -0,0 +1,136 @@
+# Script to test the different template combinations
```

Comment:
This is just the powershell script that I'm using to programatically generate all versions of the template. I will remove it later.